microprofile-fault-tolerance/src/test/java/org/wildfly/quickstarts/microprofile/faulttolerance/CoffeeResourceIT.java

```diff
@@ -17,28 +17,47 @@
 package org.wildfly.quickstarts.microprofile.faulttolerance;
 
 import java.util.List;
+import java.net.URL;
+import java.util.ArrayList;
+import jakarta.inject.Inject;
 import jakarta.ws.rs.client.Client;
 import jakarta.ws.rs.client.ClientBuilder;
 import jakarta.ws.rs.core.GenericType;
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
 import org.jboss.resteasy.plugins.providers.RegisterBuiltin;
 import org.jboss.resteasy.plugins.providers.jackson.ResteasyJackson2Provider;
 import org.jboss.resteasy.spi.ResteasyProviderFactory;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 
 /**
  * Simple integration test for Coffee resource that is guarded by MicroProfile Fault Tolerance annotations.
  *
  * @author Radoslav Husar
  * @author Eduardo Martins
  */
+
+@RunWith(Arquillian.class)
 public class CoffeeResourceIT {
 
+    private static final String APP_NAME = "microprofile-fault-tolerance";
+
+    @ArquillianResource
+    private URL deploymentUrl;
+
+    @Inject
+    private CoffeeResource coffeeResource;
+
+    @Inject
+    CoffeeRepositoryService coffeeRepository;
```

Comment:
Remove this as this won't work when testing against deployment running in e.g. OpenShift.